google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -43,8 +44,8 @@
   private JLabel warningMessage;
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
 
-  private static final String CLOUD_SDK_EXECUTABLE_PATH = "/a/b/c/gcloud-sdk/bin/gcloud";
-  private static final String CLOUD_SDK_DIR_PATH = "/a/b/c/gcloud-sdk";
+  private static final String CLOUD_SDK_EXECUTABLE_PATH = new File("/a/b/c/gcloud-sdk/bin/gcloud").getAbsolutePath();
```

Comment:
This might work too: "/a/b/c/gcloud-sdk/bin/gcloud".replace("/", File.pathSeparator)